### PR TITLE
7 course page

### PR DESCRIPTION
**Pull request by JohanAanesen**
* Front end tab action ohh 💦 
* Sexy github-flavored MarkDown to HTML 🍆 💦 💦
* Currently static assignment-page //todo
* 🤓 🍆 💦

### CODE REVIEW

File: template/course.tmpl [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{{define "title"}}Course{{end}}
+{{define "head"}}{{end}}
+{{define "content"}}
+<main role="main" class="container">
+
+    <style>
```
Comment (kongebra) [inline template/course.tmpl]: Can be moved to "head"

**Comment by JohanAanesen** [comment]
[#7 - Course page](https://trello.com/c/NgYgxRsH/8-7-course-page)

File: shared/db/userdb.go [hunk 2]
```diff
@@ -124,3 +124,28 @@ func GetHash(id int) string {
 
 	return ""
 }
+
+func IsParticipant(userID int, courseID int) bool {
```
Comment (BredeFK) [inline shared/db/userdb.go]: This function already exist in course.db as UserExistsInCourse, either remove the old one or switch

File: shared/db/coursedb.go [hunk 3]
```diff
@@ -48,6 +48,45 @@ func GetCoursesToUser(userID int) model.Courses {
...
+		var description string
+		var year string
+		var semester string
+
+		rows.Scan(&id, &courseCode, &courseName, &teacher, &description, &year, &semester)
+
+		// Add course to courses array
+		course = model.Course{
+			ID:          id,
+			Code:        courseCode,
```
Comment (BredeFK) [inline shared/db/coursedb.go]: Maybe add to the course.Hash to

File: shared/db/userdb.go (hunk 2 above)
Comment (JohanAanesen) [reply shared/db/userdb.go]: fixd

File: shared/db/coursedb.go (hunk 3 above)
Comment (JohanAanesen) [reply shared/db/coursedb.go]: fixd

**Pull request merged**